src/bindings/jna/libelektra/build.gradle

```diff
@@ -0,0 +1,57 @@
+// apply project configuration for java libraries
+apply from: "$rootProject.projectDir/java-library.gradle"
+
+// apply project dependencies
+apply from: 'dependencies.gradle'
+
+// apply maven publish configuration
+apply from: "$rootProject.projectDir/maven-publish.gradle"
+
+// confugure maven-publish plugin
+publishing {
+	publications {
+		libelektra(MavenPublication) {
+			groupId rootProject.groupId
+			artifactId 'libelektra'
+			version project.version
+
+			from components.java
+
+			pom {
+				name = "$project.name - $parent.name"
+				description = 'Low-level Elektra java binding to elekrify Java applications and write Elektra plugins using Java.'
+				url = 'https://www.libelektra.org'
+
+				licenses {
+					license {
+						name = 'BSD 3-Clause License'
+						url = 'https://github.com/ElektraInitiative/libelektra/blob/master/LICENSE.md'
+					}
+				}
+
+				// TODO #3731 - add developers responsible for code
+				developers {
+					developer {
+						id = 'libelektra'
+						name = 'Elektra Initiative'
+						email = 'elektra@libelektra.org'
+					}
+				}
```

Comment:
This part is required for the transition from the staging to release repository.
The [documentation](https://maven.apache.org/pom.html#developers) states that only the developers directly responsible for this code should be  mentioned.
It is also rather uncommon to list an organization as developer.
@markus2330 which people should we add here?

Reply:
Simply list @tucek as developer and @kodebach, me and you as contributor. (Or also as devs, however they see themselves.)

Generating the XML out of doc/AUTHORS.md seems to be over-the-top.

Reply:
> Generating the XML out of doc/AUTHORS.md seems to be over-the-top.

That is actually pretty much the opposite of what the docs suggest. To quote: "A good rule of thumb is, if the person should not be contacted about the project, they do not need to be listed here."

> It is also rather uncommon to list an organization as developer.

I don't think it is that uncommon. Especially, if a package is developed by a company.

Examples:

Android Libraries from Google just list "The Android Open Source Project" as developer. https://mvnrepository.com/artifact/androidx.activity/activity/1.2.2 

Official Kotlin Libraries also list various organizations:
https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib/1.5.0-M2
https://mvnrepository.com/artifact/io.ktor/ktor-client-core/1.5.3
https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.4.3

Log4J doesn't list any developers and instead just defines two mailing lists:
https://mvnrepository.com/artifact/log4j/log4j/1.2.17

I think the most important part is that people know how to get in touch, if there is a problem. To that end listing `elektra@libelektra.org` is probably a good idea. Especially, since the people involved in Elektra could change.

Reply:
> That is actually pretty much the opposite of what the docs suggest. To quote: "A good rule of thumb is, if the person should not be contacted about the project, they do not need to be listed here."

I do not know what you mean, doc/AUTHORS.md is exactly the list of people that are to be contacted (for their respective modules). We do not distinguish devs and contributors, though.

> If it's also okay for @markus2330 then I will just keep the organization as the only developer.

Yes, if they allow it, I would prefer it. It is hard enough to maintain doc/AUTHORS.md. E.g. @tucek is not listed there and @robaerd only states "debian packaging" but "cpack packaging" or similar (next to other things like Docker images) would be more appropriate. Some people without any code in the current repo are also to be removed. I added the task to  #3746

Reply:
> I do not know what you mean, doc/AUTHORS.md is exactly the list of people that are to be contacted (for their respective modules).

AFAIK `doc/AUTHORS.md` is supposed to list all people that contributed to Elektra (that's not the case, but that was the idea I think). I wouldn't really want to be listed as a point of contact, just because I wrote a few lines of code somewhere. Listing the general contact info makes a lot more sense. Often somebody other than the person who wrote the code might be able to help as well.

Also, by generating the Maven authors list from `AUTHORS.md` we may run into out-of-sync issues. What if we publish the Maven package today and tomorrow `AUTHORS.md` changes?

Reply:
> AFAIK doc/AUTHORS.md is supposed to list all people that contributed to Elektra (that's not the case, but that was the idea I think). I wouldn't really want to be listed as a point of contact, just because I wrote a few lines of code somewhere.

These people also don't get added.

> Listing the general contact info makes a lot more sense.

As most contact happens via the issue tracker, doc/AUTHORS.md is rarely used anyway.

> Also, by generating the Maven authors list from AUTHORS.md we may run into out-of-sync issues.

Shouldn't have mentioned the idea to generate the authors list :wink: